pkg/apis/servicecatalog/validation/broker.go

```diff
@@ -90,3 +90,10 @@ func ValidateBrokerUpdate(new *sc.Broker, old *sc.Broker) field.ErrorList {
 	// allErrs = append(allErrs, validateBrokerStatusUpdate(new, old)...)
 	return allErrs
 }
+
+// ValidateBrokerStatusUpdate checks that when changing from an older broker to a newer broker is okay.
+func ValidateBrokerStatusUpdate(new *sc.Broker, old *sc.Broker) field.ErrorList {
+	allErrs := field.ErrorList{}
+	allErrs = append(allErrs, ValidateBrokerUpdate(new, old)...)
+	return allErrs
+}
```

Comment:
Isn't this function not adding anything to what `ValidateBrokerUpdate` already checks? Why, then, carve out a separate function for this?